### ISSUE TITLE
No longer works in 10.11

### ISSUE THREAD

**Issue by tomlube** [puffnfresh/toggle-osx-shadows#17]
Not expecting a fix anytime soon, but this no longer works :(

**Comment by sammcj** [comment]
Confirmed.

**Comment by OskarGroth** [comment]
WindowServer: Untrusted Client attempted to set debug options

Some more info on "trusted clients" here: 
http://stackoverflow.com/questions/18603055/getting-a-mac-to-trust-an-application-synergy-on-loginwindow

**Comment by vikohone** [comment]
Confirmed.

**Comment by akobrin** [comment]
Confirmed.

**Comment by alexwebbb** [comment]
Confirmed.

**Comment by alex-humphreys** [comment]
Confirmed.

**Comment by CouncilOne** [comment]
Confirmed.

**Comment by jacksonmcclelland** [comment]
Confirmed.

**Comment by ross-w** [comment]
Confirmed.

**Comment by alex-humphreys** [comment]
Confirmed.

**Comment by puffnfresh** [comment]
Confirmed.

**Comment by phantompi** [comment]
Confirmed.

**Comment by leifniem** [comment]
Confirmative.

**Comment by ghost** [comment]
Confirmed

**Comment by tomlube** [comment]
What the fuck guys lmao I thought I kept getting emails about the same comment going all the way back to August I was so annoyed with github

**Comment by sammcj** [comment]
Confirmed you're not getting emails about the same comment.

**Comment by nmcv** [comment]
Confirmed all.

**Comment by evanpurkhiser** [comment]
I'm sure more than a few people here would be more than happy to disable system integrity protection system (no matter how irresponsible that likely is). I wonder what kind of modifications need to be made to this software to have it working again, if it's at all possible.

**Comment by victorluft** [comment]
I haven't yet seen any way to get CGSDebug working on El Capitan, but [it is possible](https://github.com/norio-nomura/EasySIMBL/issues/26#issuecomment-117028426) to get SIMBL working with SIP enabled, so if you want to make a SIMBL bundle that sets `hasShadow` to false on NSWindows (with whatever toggling logic you want) that'll work.

**Comment by snario** [comment]
:(

**Comment by joseph-tran-tn** [comment]
Waiting..... :(

**Comment by 0ldM4cM4n** [comment]
Bummer! I just installed clib and toggle-osx-shadows, only to discover -- like many others here -- that it no longer works with 10.11.4, or apparently with earlier iterations of 10.11 either. Sometimes Apple's heavy-handedness really irks me. Considering how much we pay for our machines, why can't we have the option to decide whether or not we want drop shadows under our desktop windows?

And don't get me going about the current, very drab, almost-colorless, gray color scheme. Blah! :(

**Comment by noshery** [comment]
> if you want to make a SIMBL bundle that sets hasShadow to false on NSWindows (with whatever toggling logic you want) that'll work

Thanks victorluft for the idea. I was wondering if you or anyone else could help with a few questions I have about making a Simbl plugin to remove shadows. If I used Xcode would I need to toggle shadows for each app or just once for NSWindow? What would the address be (com.Apple.Appkit)? Do I need the NSWindow directory location? I haven't been able to find it in frameworks. I assume there isn't a .plist associated with these files to easily toggle shadows using terminal. I'm a total beginner and never used Xcode before.

**Comment by towerofnix** [comment]
Confirmed (with SIP disabled).

**Comment by vigo** [comment]
doesn't work :(

**Comment by fregante** [comment]
There's no need to comment any more, everybody knows that it doesn't work.

If you want to be notified in case that changes, click the `[Subscribe]` button on the right (or bottom)

**Comment by towerofnix** [comment]
> There's no need to comment any more, everybody knows that it doesn't work.

To show people would still like to use this, should we maybe be using GitHub reactions (top right corner of post) on the first post? :)

**Comment by fregante** [comment]
Yes, reactions were added to avoid polluting comments with `+1`'s

**Comment by sammcj** [comment]
@puffnfresh - want to lock this thread? I'm getting sick of notifications from +1's but want to know if it ever gets fixed?
